test/build_apache.sh

```diff
@@ -81,14 +81,17 @@ popd
 # get built image name
 # only match the first line
 SNOWFLAKE_PLUGIN_NAME=$(ls $SNOWFLAKE_PLUGIN_PATH | grep "$SNOWFLAKE_PLUGIN_NAME_REGEX" | head -n 1)
-echo -e "\n=== built connector name: $SNOWFLAKE_PLUGIN_NAME ==="
+echo -e "\n========================= BUILD COMPLETE ========================="
+echo -e "built connector name: $SNOWFLAKE_PLUGIN_NAME"
 
 # copy built connector to plugin path
 mkdir -m 777 -p $KAFKA_CONNECT_PLUGIN_PATH || \
 sudo mkdir -m 777 -p $KAFKA_CONNECT_PLUGIN_PATH 
 cp $SNOWFLAKE_PLUGIN_PATH/$SNOWFLAKE_PLUGIN_NAME $KAFKA_CONNECT_PLUGIN_PATH || true
-echo -e "\n=== copied connector to $KAFKA_CONNECT_PLUGIN_PATH ==="
+echo -e "copied connector to $KAFKA_CONNECT_PLUGIN_PATH"
 
 KAFKA_CONNECT_DOCKER_JAR_PATH="$SNOWFLAKE_CONNECTOR_PATH/docker-setup/snowflake-kafka-docker/jars"
 mv $SNOWFLAKE_PLUGIN_PATH/$SNOWFLAKE_PLUGIN_NAME $KAFKA_CONNECT_DOCKER_JAR_PATH || true
-echo -e "\n=== copied connector to $KAFKA_CONNECT_DOCKER_JAR_PATH for docker ==="
\ No newline at end of file
+echo -e "copied connector to $KAFKA_CONNECT_DOCKER_JAR_PATH for docker"
+
+echo -e "=== NOTE: do not share this privately built jar with customers ===\n"
```

Comment:
I think we actually do this for some special cases, usually if a feature is in PrPr and we're working with a customer on a fix, then providing a private jar is ok. The case we would want to avoid is that asking customer to run a private jar on their production environment.

Reply:
Updated the message to reflect special cases and to not let the customer run it in prod